app/assets/javascripts/modules/userChoiceSelect.js

```diff
@@ -0,0 +1,41 @@
+angular.module('probedock.userChoiceSelect', [ 'probedock.api' ])
+  .directive('userChoiceSelect', function() {
+    return {
+      restrict: 'E',
+      controller: 'UserChoiceSelectCtrl',
+      templateUrl: '/templates/user-choice-select.html',
+      scope: {
+        organization: '=',
+        userIds: '=ngModel'
```

Comment:
tried: `'model.userIds': '=ngModel'` but it does not work. Therefore, I use an alias with `userIds` and a `$watch` to update it from `model.userIds` which is used by ui-select.